### PR TITLE
Move `parsers.js` to pkg/lib - SQUASH ON LANDING

### PR DESCRIPTION
**Pull request by MahmoudHamdy02**
The `parsers.js` file is needed in Navigator for editing permissions
https://github.com/KKoukiou/cockpit-navigator/pull/54

### CODE REVIEW

**Review by KKoukiou — APPROVED** [review]

File: pkg/users/users.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ import { createRoot } from 'react-dom/client';
 import { superuser } from "superuser";
 
 import { usePageLocation, useLoggedInUser, useFile, useInit } from "hooks.js";
-import { etc_passwd_syntax, etc_group_syntax, etc_shells_syntax } from "./parsers.js";
+import { etc_passwd_syntax, etc_group_syntax, etc_shells_syntax } from "parsers.js";
```
Comment (martinpitt) [inline pkg/users/users.js:29]: Please rename the file then -- "parsers.js" is too generic as a public API. Perhaps `nss_parser` or `pam_user_parser.js` ?

**Review by martinpitt — CHANGES_REQUESTED** [review]
See above

File: pkg/users/users.js (hunk 1 above)
Comment (MahmoudHamdy02) [reply pkg/users/users.js]: @martinpitt done :)

**Review by martinpitt — APPROVED** [review]
Thanks!

**Pull request merged**